source/install.txt

```diff
@@ -218,7 +222,10 @@ Select your operating system:
       Install Compass
       ---------------
 
-      1. Double-click the installer file.
+      1. Ensure that you are running PowerShell or your Azure desktop as
+         an administrator.
+         
+      #. Double-click the installer file.
 
```

Comment:
A similar comment applies here. Should this be more of a note at the beginning of the procedure saying:

"If you are installing using PowerShell or your Azure desktop, ensure that you are running as an administrator."

or similar.

Also, I would think that if we are assuming users are in PowerShell, we would provide a PowerShell command rather than telling them to double click the installer file in step 2? Since double-clicking the installer file would just start the .msi, and not use PowerShell at all.

Reply:
Yes, this content is not needed here at all, it only is needed in the prereqs. Will remove from here.